packages/bierzo-wallet/src/components/Wallet/ContinueButton/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button, withStyles } from '@material-ui/core';
+import React from 'react';
+
+const StyledButton = withStyles({
+  root: {
+    '&&': {
+      outline: '1px solid red',
+      width: '75%',
+      height: '42px',
+      borderRadius: '4px',
+      backgroundColor: '#31e6c9',
+      color: '#ffffff',
+    },
+  },
+  label: {
+    '&&': {
+      textTransform: 'capitalize',
+    },
+  },
+})(Button);
+
+const ContinueButton = (props: { positioning: React.CSSProperties }) => {
```

Comment:
Using Typescript you should declare types in this way:
```
interface Props {
  readonly positioning: React.CSSProperties;
}

....

const continueButton = ({ positioning }: Props) => {
    ....
```

Reply:
Fixed by [3daf67f](https://github.com/iov-one/ponferrada/pull/79/commits/3daf67fea9627053c6c04109b97b5e0b2e8a2b46).